generators/cleanup.js

```diff
@@ -118,6 +118,7 @@ function cleanupOldFiles(generator) {
             generator.removeFile(`${ANGULAR_DIR}shared/login/login.component.ts`);
             generator.removeFile(`${ANGULAR_DIR}shared/login/login.component.html`);
             generator.removeFile(`${ANGULAR_DIR}core/auth/user-route-access-service.ts`);
+            generator.removeFile(`${ANGULAR_DIR}core/auth/csrf.service.ts`);
```

Comment:
You shouldn’t do this way, 2 options here:
- remove when not needed
- use `this.fs.delete(this.destinationPath(...));`

Otherwise any change to the file will be overwritten every regeneration.

Reply:
cc @kaidohallik

Reply:
OK, adding condition, will do PR shortly.